scripts/requirements-ci.txt

```diff
@@ -15,3 +15,4 @@ stringcase==1.2.0
 toml
 wget
 python-dotenv
+ruff
```

Comment:
why is this needed ?

I thought we were already installing all requirements from https://github.com/nrfconnect/sdk-zephyr/blob/main/scripts/requirements.txt which includes https://github.com/nrfconnect/sdk-zephyr/blob/main/scripts/requirements-compliance.txt#L14
```
ruff==0.11.11
```

@PerMac do we really need to duplicate all pip requirements which are already specified in `sdk-zephyr/scripts/requirements-compliance.txt` ?

Reply:
Changes dropped